inc/default-styles.php

```diff
@@ -65,65 +65,50 @@ static function register_scripts() {
 		) );
 	}
 
-	static function row_style_fields( $fields ) {
-		// Add the attribute fields
+	/**
+	 * These are general styles that apply to all elements
+	 *
+	 * @param $id
+	 * @param $label
+	 *
+	 * @return array
+	 */
+	static function get_general_style_fields( $id, $label ) {
+		$fields = array();
+
+		// All the attribute fields
 
 		$fields['id'] = array(
-			'name'        => __( 'Row ID', 'siteorigin-panels' ),
+			'name'        => sprintf( __( '%s ID', 'siteorigin-panels' ), $label ),
 			'type'        => 'text',
 			'group'       => 'attributes',
-			'description' => __( 'A custom ID used for this row.', 'siteorigin-panels' ),
+			'description' => sprintf( __( 'A custom ID used for this %s.', 'siteorigin-panels' ), strtolower( $label ) ),
```

Comment:
Technically in all the cases we're using it for, yeah. Although the main difference between the label and the ID is spaces. So say we add some different type of _thing_ to Page Builder that requires a 2 word label. The ID would be `some-thing` and the label would be `Some thing`.